### PR TITLE
Adding new language Hungarian Beta

### PR DESCRIPTION
**Pull request by ctlaltdieliet**
```release-notes
Added a new translation Hungarian (beta)
```

### CODE REVIEW

File: app/i18n/index.js [hunk 1]
```diff
@@ -28,6 +28,11 @@ function loadTranslation(locale) {
             localeData = require('react-intl/locale-data/de');
             momentData = require('moment/locale/de');
             break;
+        case 'en-AU':
```
Comment (metanerd) [inline app/i18n/index.js:31]: I guess this should still be removed.

Comment (ctlaltdieliet) [reply app/i18n/index.js]: I started in the wrong branch on my local machine.  Removed en-AU

**Review by enahum — APPROVED** [review]

**Review by josephbaylon — APPROVED** [review]
LGTM

**Comment by metanerd** [comment]
> Thanks!
> 
> @ctlaltdieliet could we use this same approach for English-Australian? https://github.com/mattermost/mattermost-mobile/pull/5237/files

No, the same approach cannot be used.

**Pull request merged**